src/netex_france/exporter.rs

```diff
@@ -24,16 +29,19 @@ use std::{
 };
 
 const NETEX_FRANCE_STOPS_FILENAME: &str = "arrets.xml";
+const NETEX_FRANCE_LINES_FILENAME: &str = "lignes.xml";
 
 enum VersionType {
     Stops,
+    Lines,
 }
 
 impl Display for VersionType {
     fn fmt(&self, fmt: &mut Formatter) -> std::result::Result<(), fmt::Error> {
         use VersionType::*;
         match self {
             Stops => write!(fmt, "ARRET"),
+            Lines => write!(fmt, "LINE"),
```

Comment:
It should be "LIGNE", in french (specs). No ?

Reply:
You're right, nice catch. Fixed in 01dd9729c2ef7bf03c17d93407cdf679a00dcb5e.